src/backends/win32.rs

```diff
@@ -137,10 +137,10 @@ impl Buffer {
 /// The handle to a window for software buffering.
 pub struct Win32Impl<D: ?Sized, W> {
     /// The window handle.
-    window: HWND,
+    window: OnlyUsedFromOrigin<HWND>,
```

Comment:
Isn't `HWND` and `HDC` thread-safe (so they could be used from any origin), as you've also realized with `unsafe impl Send for Command {}`?